internal/compiler/passes/ensure_window.rs

```diff
@@ -32,26 +32,11 @@ pub fn ensure_window(
     let new_root = Element {
         id: std::mem::replace(&mut win_elem_mut.id, "root_window".into()),
         base_type: std::mem::replace(&mut win_elem_mut.base_type, window_type),
-        bindings: Default::default(),
-        property_analysis: Default::default(),
         children: std::mem::take(&mut win_elem_mut.children),
         enclosing_component: win_elem_mut.enclosing_component.clone(),
-        property_declarations: Default::default(),
-        named_references: Default::default(),
-        repeated: Default::default(),
-        states: Default::default(),
-        transitions: Default::default(),
-        child_of_layout: false,
-        has_popup_child: false,
-        layout_info_prop: Default::default(),
-        default_fill_parent: Default::default(),
-        accessibility_props: Default::default(),
-        is_flickable_viewport: false,
-        item_index: Default::default(),
-        item_index_of_first_children: Default::default(),
         node: win_elem_mut.node.clone(),
         inline_depth: 0,
-        is_legacy_syntax: false,
+        ..Default::default()
```

Comment:
i think i didn't do that before because i actually want to have an error when adding new fields. But i guess it's fine in this case